docs/tutorials/developer-journey/level-0/03-dev-env.mdx

```diff
@@ -120,15 +120,15 @@ In future modules, you'll refer to this directory as the **working directory**.
 
 #### Why does this matter?
 
-You'll use this working directory to contain the projects that you build throughout your developer journey. This will help keep things organized in our local file structure.
+You'll use this working directory to contain the projects that you build throughout your developer ladder. This will help keep things organized in our local file structure.
```

Comment:
```suggestion
You'll use this working directory to contain the projects that you build throughout this series. This will help keep things organized in your local file structure.
```